contents/docs/self-host/deploy/deployment-sizing-faq.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Deploymet Sizing FAQ
+sidebar: Docs
+showTitle: true
+---
+
+We are often asked what kind of configuration is needed to run PostHog at 10M / 50M / 100M / 1B /... events per month. In practice, this is a difficult question to answer because of the large amount of variables affecting performance. Here is some context on how to think about this in the form of "frequently asked questions" which might help. If you need additional clarification, feel free to ask questions in the relevant documentation pages directly.
+
+
+## Can I use PostHog in production without kubernetes
+
+Our [hobby deployment](/docs/self-host/deploy/hobby) does not use kubernetes, but is not set up to handle production volumes. It is not currently possible to deploy a production-ready PostHog instance without kubernetes.
+
+## Kubernetes: cluster replication / node count
+
+We do not yet provide replicated/sharded datasets for self hosted installations, so it's not recommended to have more than 1 node per stateful service (1 Redis node, 1 Kafka node, 1 ClickHouse node, ...). In the future, we will provide the possibility to share and balance loads across multiple components in the same tier (example: N nodes handle Kafka traffic) but currently that’s not an option.
+
+Generally, one node per stateful services is needed. At max, you should account for 1 node per stateful services + N nodes for the stateless services (i.e., PostHog itself; N will be between 2 and 20). All stateful services are currently down when the related Kubernetes pod goes down. We then restore the service when Kubernetes reschedules and restarts the pod (the dataset is safe as we persist it on a dedicated disk that gets re-attached when the pod gets rescheduled).
```

Comment:
This repeats what you say above with a better explanation, I'd suggest condensing this into one paragraph capturing what's in your second point here but also that you can't currently have replication or sharding so hence 1 node max for the stateful services is what you need.